### PR TITLE
Sign Up Functinality addded

### PR DESCRIPTION
**Pull request by kabirulislam95**

### CODE REVIEW

File: techeria_controller/techeria_app/models.py [hunk 1]
```diff
@@ -20,16 +20,16 @@ class Meta:
 class BuyerModel(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
-    user_name = models.CharField(max_length=255)
+    username = models.CharField(max_length=255)
     date_of_birth = models.DateField()
     email = models.EmailField()
-    mobile_number = models.IntegerField(max_length=10)
+    mobile_number = models.IntegerField()
```
Comment (farahh001) [inline techeria_controller/techeria_app/models.py:26]: we can use the phonenumber_field library as we did for email. what do you think

File: techeria_controller/techeria_app/models.py [hunk 2]
```diff
@@ -38,15 +38,19 @@ class Meta:
 class SellerModel(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
-    user_name = models.CharField(max_length=255)
+    username = models.CharField(max_length=255)
     date_of_birth = models.DateField()
     email = models.EmailField()
-    mobile_number = models.IntegerField(max_length=10)
```
Comment (farahh001) [inline techeria_controller/techeria_app/models.py:44]: Here also we can add phonenumber_field library

File: techeria_controller/techeria_app/views.py [hunk 3]
```diff
@@ -71,43 +71,144 @@ def product(request):
...
+from django.http import response
+from django.shortcuts import redirect, render
+from techeria_app.models import BuyerModel, SellerModel, Products
+from django.contrib.auth.models import User, auth
+from django.contrib import messages
 
-        elif category == "Seller":
-            seller.user_name = user_name
-            seller.first_name=first_name
-            seller.last_name=last_name
```
Comment (farahh001) [inline techeria_controller/techeria_app/views.py:95]: Did you set the verification email?

Comment (kabirulislam95) [reply techeria_controller/techeria_app/views.py:95]: ok, I will add the verification. When users want to sign up, the verification email will be sent and after sending the verification only the users will be able to sign up.

File: techeria_controller/techeria_app/models.py (hunk 2 above)
Comment (kabirulislam95) [reply techeria_controller/techeria_app/models.py:44]: got it. I will add the mobile number field for the  mobile_number

File: techeria_controller/techeria_app/models.py (hunk 1 above)
Comment (kabirulislam95) [reply techeria_controller/techeria_app/models.py:26]: Got it Farah. Thanks for your participation

**Pull request merged**